dist/api.web-bluetooth.js

```diff
@@ -499,14 +499,14 @@
             adapter.enableNotify(this._handle, function(dataView) {
                 this.value = dataView;
                 this.dispatchEvent({ type: "characteristicvaluechanged", bubbles: true });
-            }.bind(this), resolve, wrapReject(reject, "startNotifications error"));
+            }.bind(this), resolve(this), wrapReject(reject, "startNotifications error"));
```

Comment:
This executes resolve when calling `enableNotify()` instead of passing the resolve function to be executed by `enableNotify()`